pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools >= 40.6.0", "wheel"]
+build-backend = "setuptools.build_meta"
```

Comment:
Can we add a line change here?

Reply:
@villebro do you mean adding an empty line at the end of the file?

Reply:
@leo-schick IIRC, that red dash with a circle around it means the last line hasn't been terminated with a line change:
<img width="471" alt="image" src="https://user-images.githubusercontent.com/33317356/214098397-70ca9299-b8cc-4926-8cd6-495fbec41405.png">

Reply:
It might also need the apache license boilerplate, i.e.
```
#
# Licensed to the Apache Software Foundation (ASF) under one or more
# contributor license agreements.  See the NOTICE file distributed with
# this work for additional information regarding copyright ownership.
# The ASF licenses this file to You under the Apache License, Version 2.0
# (the "License"); you may not use this file except in compliance with
# the License.  You may obtain a copy of the License at
#
#    http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
```

Reply:
Done.